### ISSUE TITLE
"INVITER EN VENN TIL Å GI SITT BIDRAG" Button nach Submit

### ISSUE THREAD

**Issue by diplomod** [diplomod/nordryggstien.no#204]
Unter den beiden Buttons, die wir jetzt schon haben nach der Submit-Seite, brauchen wir noch einen weiteren, welcher sagt: "INVITER EN VENN TIL Å GI SITT BIDRAG" - da soll sich dann eine Message-Form öffnen, in der der Einladende eine Email-Adresse eines Freundes eing

Betreff "Del et geofunn langs Nordryggstien!"

Text:
nordryggstien.no består av en rekke geofunn. Vi ønsker bidrag til nye geofunn fra geologer og andre som er interesert i geologi. Hensikten er at dette skal bli en evigvarende stafett. Vi gir deg anledning til å promotere din egen favorittlokalitet.

For at vi skal få mange nye geofunn er det fint om du sender inn ditt geofunn innen en til to uker, og samtidig inviterer en venn til å gi sitt bidrag. Du må gjerne invitere flere venner.

Les mer om Nordryggstien og legg inn ditt geofunn på http://www.nordryggstien.no."

Text soll noch personalisierbar sein.

**Comment by lukassiedel** [comment]
"Text soll noch personalisierbar sein." für den admin oder für den Nutzer selber bevor er abschickt?

**Comment by diplomod** [comment]
Für den Nutzer, der abschickt. :) Wenn wir im Admin den Text administrieren können, super. Wenn der so wie angegeben fix drinsteht auch recht.

**Comment by lukassiedel** [comment]
Da die Seite mit den Buttons nach dem submitten immoment gar nicht genutzt wird, ist das noch relevant?

**Comment by diplomod** [comment]
Ja, aber siehe Milestone - erst min der nächsten Runde, aktuell nicht.

**Comment by lukassiedel** [comment]
Ok soll also iwann wieder kommen. Nur weil wir grad an den Schätzungen sind
